blocks/cards/cards.css

```diff
@@ -246,6 +255,12 @@ main .cards-wrapper .machine-parts .cards-card-body h3{
     padding-right: 0;
   }
 
+  .cards.grid > ul {  
+    display: grid;
```

Comment:
The `cards` [block in the bolierplate ](https://github.com/adobe/aem-boilerplate/blob/main/blocks/cards/cards.css) is also a `grid` so I think its good we are doing the same. 👍 
On the same lines, can it be made the default? Or is there a use case you think needs a `non-grid` cards block?

Reply:
@sdmcraft We have another two cases that are using cards, they use Flex, and the image size, margin, padding, etc.. are different. If I make grid cards as default, we need to create a variant for them. So I think it's good to keep it as is. What do you think?

1. Related articles(3 columns)

<img width="1234" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/6a08d33f-1be5-4a9c-afce-fa640e562b08">


2. Featured articles(4 columns)
<img width="1224" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/90106485-edf6-476d-a7b4-1a960cf026cb">